packages/connectivity/src/scp-cf/environment-accessor.ts

```diff
@@ -87,7 +87,7 @@ export function getServiceCredentialsList(service: string): any[] {
  * @internal
  */
 export function getServiceList(service: string): Service[] {
-  return xsenv.filterServices({ label: service });
+  return xsenv.filterCFServices({ label: service });
```

Comment:
Does the old function not exist?
The naming suggests that on k8s this won't work correctly.